### PR TITLE
NumPy course updates, July 2018

### PR DESCRIPTION
**Pull request by dkillick**
Improve a few sections of the NumPy course - correcting a couple of typos, clarifying one exercise, and adding a couple of missing doc links.

### CODE REVIEW

**Comment by dkillick** [comment]
Ping @kaedonkers

File: course_content/notebooks/numpy_intro.ipynb [hunk 1]
```diff
@@ -1147,9 +1147,8 @@
     "#### Exercise\n",
     "\n",
     "  * Create a masked array from the numbers 0-11, where all the values less than 5 are masked.\n",
-    "  * How can you create masked data from an array of positive values where a value of `-1.0` represents 'missing'?\n",
-    "  * What special masked array creation routines exist to do this kind of thing more efficiently?\n",
-    "    * HINT: look up `np.ma.masked_where` and related routines.\n",
+    "  * Create a masked array of of positive values, with a value of `-1.0` to represent missing points.\n",
```
Comment (kaedonkers) [inline course_content/notebooks/numpy_intro.ipynb]: One too many **of**'s

**Comment by pelson** [comment]
@dkillick - I've just added @kaedonkers to the maintainers of this and scitools.org.uk, so hopefully you should be able to get a timely review **and** somebody to merge 😄 🎉.

Thanks @kaedonkers for providing good quality reviews of the material. Keep up the great work! 👍

**Comment by kaedonkers** [comment]
Thanks @pelson!

**Pull request merged**